client/src/partials/projects/projects.html

```diff
@@ -1,21 +1,17 @@
-<header data-header>
-  {{ 'PROJECT.TITLE' | translate }}
-</header>
-
-<nav>
-  <div class="pull-left">
-    <ol class="breadcrumb">
-      <li><a href="#/"><span class="glyphicon glyphicon-home"></span></a></li>
+<div class="flex-header">
+  <div class="bhima-title">
+    <ol class="headercrumb">
+      <li class="static">{{ "TREE.ADMIN" | translate }}</li>
       <li class="active">{{ "PROJECT.TITLE" | translate }}</li>
     </ol>
   </div>
+</div>
 
-  <div class="pull-right">
-    <button class="btn btn-sm btn-default" ng-click="new()">
-      <span class="glyphicon glyphicon-plus-sign"></span> {{ 'PROJECT.NEW' | translate }}
-    </button>
-  </div>
-</nav>
+<div class="flex-util">
+  <button class="btn btn-sm btn-default" id="create" ng-click="ProjectCtrl.create()" data-method="create">
+    <span class="glyphicon glyphicon-plus-sign"></span> {{ 'PROJECT.NEW' | translate }}
+  </button>
+</div>
 
 <div class="report visible-print">
```

Comment:
You can delete this entire section (`<div class="report visible-print>"`) - it is no longer used and is bad practice. It also slows down the loading of the module.